server/command.go

```diff
@@ -143,6 +154,17 @@ func getAutoCompleteData() *model.AutocompleteData {
 	subscribe := model.NewAutocompleteData("subscribe", "", "Subscribe the current channel to notifications from Confluence")
 	confluence.AddCommand(subscribe)
 
+	config := model.NewAutocompleteData("config", "", "Config related options for confluence instances")
+	configOptions := []model.AutocompleteListItem{{
+		HelpText: "Add config for the confluence instance",
+		Item:     "add",
+	}, {
+		HelpText: "List all the saved configs",
+		Item:     "list",
+	}}
+	config.AddStaticListArgument("", false, configOptions)
+	confluence.AddCommand(config)
+
 	unsubscribe := model.NewAutocompleteData("unsubscribe", "[name]", "Unsubscribe the current channel from notifications associated with the given subscription name")
```

Comment:
```suggestion
	unsubscribe := model.NewAutocompleteData("unsubscribe", "[name]", "Unsubscribe from the current channel from notifications associated with the given subscription name")
```

Reply:
previous is better because we are unsubscribing the current channel from the notification which are coming from a particular subscription